spec/datetimeformat.html

```diff
@@ -364,17 +364,70 @@ <h1>FormatDateTimeToParts ( _dateTimeFormat_, _x_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-tolocaltime" aoid="ToLocalTime">
-      <h1>ToLocalTime ( _date_, _calendar_, _timeZone_ )</h1>
+      <h1>ToLocalTime ( _t_, _calendar_, _timeZone_ )</h1>
 
       <p>
-        When the ToLocalTime abstract operation is called with arguments _date_, _calendar_, and _timeZone_, the following steps are taken:
+        When the ToLocalTime abstract operation is called with arguments _t_, _calendar_, and _timeZone_, the following steps are taken:
       </p>
 
       <emu-alg>
-        1. Apply calendrical calculations on _date_ for the given _calendar_ and _timeZone_ to produce weekday, era, year, month, day, hour, minute, second, and inDST values. The calculations should use best available information about the specified _calendar_ and _timeZone_, including current and historical information about time zone offsets from UTC and daylight saving time rules. If the _calendar_ is `"gregory"`, then the calculations must match the algorithms specified in ES2020, <emu-xref href="#sec-overview-of-date-objects-and-definitions-of-abstract-operations"></emu-xref>.
-        1. Return a Record with fields [[weekday]], [[era]], [[year]], [[month]], [[day]], [[hour]], [[minute]], [[second]], and [[inDST]], each with the corresponding calculated value.
+        1. Assert: Type(_t_) is Number.
+        1. If _calendar_ is `"gregory"`,
+          1. Let _timeZoneOffset_ be the value calculated according to <emu-xref href="#sec-local-time-zone-adjustment">LocalTZA(_t_, *true*)</emu-xref> where the local time zone is replaced with timezone _timeZone_.
```

Comment:
"where the local time zone is replaced with timezone _timeZone_" feels really strange to me. Can we update 262 to split LocalTZA into two parts?

Reply:
I thought about this -- but was unsure if it was worth addressing the vagueness of "local time" and "local time zone" references in 262's implementation defined date time algorithms.

Here is an example implementation to consider: https://github.com/tc39/ecma262/compare/master...bocoup:timezoneadjustment?expand=1#diff-3540caefa502006d8a33cb1385720803

The link above adds an abstract operation called `TimeZoneAdjustment`, which is `LocalTZA` with the additional third argument `timeZone`. `LocalTZA` now calls `TimeZoneAdjustment` after setting `timeZone` to the mysterious `local time zone`.

@ljharb, do you think it's reasonable to break up ecma262's `LocalTZA` for this specific use case in ecma402? Specifically, so we can call `TimeZoneAdjustment(_t_, *true*, timeZone)` in the line @Ms2ger commented on above.

Reply:
We talked about it in an ecma402 meeting, going with the prose as is.

Reply:
I think it’d be fine to split up the abstract ops in 262, if that’d help here, for future reference.